app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,79 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+
+/* variables */
+
+$light-gray: #777;
+
+/* universal */
+
+body {
+  padding-top: 60px;
+}
```

Comment:
add blank lines between all top level styles. Do not have blank lines between nested styles.